src/test/java/net/revelc/code/formatter/html/HTMLFormatterTest.java

```diff
@@ -36,12 +36,10 @@
     void testDoFormatFile() throws Exception {
         // FIXME Handle linux vs windows since this formatter does not accept line endings
         final var expectedHash = LineEnding.LF.isSystem()
-                ? "9774ef68195bb7bd9c1d0eaed6b3c7536633d0d916ad52187005ec2c89e0fcfe42e14dd4576453cb14abaef0352ef0b7a275c015e7cc124b8b4da481f423d030"
```

Comment:
At this point, the two pass test is possible to work and was last in issue.  There is still one more bug after this that is more trivial that I have a PR out for but after 3 months I figure its going no where.